codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala

```diff
@@ -131,7 +165,7 @@ class SerializerCheckCompilerPluginComponent(
               s"""No codecs for ${actuallyMissingFullyQualifiedClassNames
                 .mkString(", ")} are registered in class annotated with @$serializerType.
                  |This will lead to a missing codec for Akka serialization in the runtime.
-                 |Current filtering regex: $filterRegex""".stripMargin)
+                 |Current filtering regex: $typeRegexPattern""".stripMargin)
```

Comment:
> filtering regex

Same in user-facing strings: pls ensure we call this regex consistently across all docs/error messages

Reply:
So, change line 168 to look like this?
```
|Current typeRegexPattern: $typeRegexPattern""".stripMargin)
```
And do the same in all other places if needed?